src/areaSearch/areaSearchLandingPage.tsx

```diff
@@ -236,19 +236,33 @@ const AreaSearchLandingPage = (): JSX.Element => {
               'Other land use arrangements',
             )}
             bottomText={
-              <p>
-                {t(
-                  'areaSearch.landingPage.otherUses.misc.blurb',
-                  'For example city farming, dog training fields, bee farming, other land use and leasing, hunting licenses, kindergarten outdoor areas, water area leasing.',
-                )}
-              </p>
+              <div>
+                <p>
+                  {t(
+                    'areaSearch.landingPage.otherUses.misc.dogs.blurb',
+                    'For example city farming, dog training fields, bee farming, other land use and leasing, hunting licenses, kindergarten outdoor areas, water area leasing.',
```

Comment:
To me it seems like the texts should be different, and there's two of them - when I look at the design page